packages/react-frontend/src/pages/home.js

```diff
@@ -162,17 +162,6 @@ function listPolls(){
   }
   
   
-    // merged tasklisthead component
-    function TaskListHead() {
-        return (
-            <thead>
-                <tr>
-                    <th>LIST OF CHORES:</th>
-                </tr>
-            </thead>
-        );
-    }
-
```

Comment:
Good job, I deleted this old code in my branch too. Remnants of when we were building off of the table that we did in the first few weeks of class.